force-app/main/default/objects/Exception__c/Exception__c.object-meta.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Adding a custom exception object is a nice touch for tracking the history of errors encountered.